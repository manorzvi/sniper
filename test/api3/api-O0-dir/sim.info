{'cmdline': ['../../run-sniper',
             '-n',
             '1',
             '-d',
             'api-O0-dir',
             '--',
             './api-O0'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (72.219235,
            0.220327,
            38360,
            0,
            0,
            0,
            8018,
            0,
            0,
            0,
            192,
            0,
            0,
            0,
            58560,
            134),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api3/api-O0-dir --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpLJX71l/run_benchmarks'],
 't_elapsed': 78.31825184822083,
 't_start': 1633352470.906073,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpLJX71l/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api-O0']],
 'user': 'root',
 'vmem': 220901376L}
