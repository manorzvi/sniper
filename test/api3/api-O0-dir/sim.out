                                     | Core 0    
  Instructions                       |   50121291
  Cycles                             |  120288658
  IPC                                |       0.42
  Time (ns)                          |   45221300
  Idle time (ns)                     |        -12
  Idle time (%)                      |      -0.0%
Branch predictor stats               |           
  num correct                        |   10021903
  num incorrect                      |       1509
  misprediction rate                 |      0.02%
  mpki                               |       0.03
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |      15293
    num misses                       |         58
    miss rate                        |      0.38%
    mpki                             |       0.00
  D-TLB                              |           
    num accesses                     |   60040217
    num misses                       |        105
    miss rate                        |      0.00%
    mpki                             |       0.00
  L2 TLB                             |           
    num accesses                     |        163
    num misses                       |        129
    miss rate                        |     79.14%
    mpki                             |       0.00
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |      15293
    num cache misses                 |        892
    miss rate                        |      5.83%
    mpki                             |       0.02
  Cache L1-D                         |           
    num cache accesses               |   60040217
    num cache misses                 |       1796
    miss rate                        |      0.00%
    mpki                             |       0.04
  Cache L2                           |           
    num cache accesses               |       2725
    num cache misses                 |       2482
    miss rate                        |     91.08%
    mpki                             |       0.05
  Cache L3                           |           
    num cache accesses               |       2521
    num cache misses                 |       2480
    miss rate                        |     98.37%
    mpki                             |       0.05
DRAM summary                         |           
  num dram accesses                  |       2480
  average dram access latency (ns)   |      72.77
  average dram queueing delay        |      19.34
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |       1113
  num loads from dram cache          |          0
  num loads from remote cache        |          0
